rpc/clients/typescript/src/ws_client.ts

```diff
@@ -287,36 +287,50 @@ export class WSClient {
     /**
      * Get all 0x signed orders currently stored in the Mesh node
```

Comment:
Can we cherry-pick the updates to the ws client tests onto this branch? It seems like we should make sure our best tests get brought onto our release branches.

Reply:
Unfortunately those tests use features in the V3 0x libraries that don't exist in V2 and cherry-picking them would involve re-writing a bunch of the test setup. We agreed offline to leave this as is.